src/client/Microsoft.Identity.Client/Internal/Broker/NullBroker.cs

```diff
@@ -64,6 +64,12 @@ public Task<MsalTokenResponse> AcquireTokenSilentDefaultUserAsync(Authentication
             return Task.FromResult<MsalTokenResponse>(null);
         }
 
+        public Task<MsalTokenResponse> AcquireTokenByUsernamePasswordAsync(AuthenticationRequestParameters authenticationRequestParameters, AcquireTokenSilentParameters acquireTokenSilentParameters)
```

Comment:
I think this overload is not needed. I don't see an overload that accepts a acquireTokenSilentParameters parameter. Also, not sure this flow needs these parameters anyway as they are for silent requests?? I think the logic should live in the method on line 84.